### DISCUSSION TITLE
Walls appears damaged in transparent/translucent LOS mode

### DISCUSSION THREAD

**Discussion by SomeRandomNoobKekeke** [FakeFishGames/Barotrauma#13769] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

It's clearly visible in biomes with bright ambient light, like hydro wastes
It's a bit ugly and misleading so i guess it's not very intended
![20240412135827_1](https://github.com/FakeFishGames/Barotrauma/assets/122838333/41b5a94c-28c2-491f-8244-b0da4caeb629)

![20240412140708_1](https://github.com/FakeFishGames/Barotrauma/assets/122838333/10395833-232a-4826-a9cc-bda5841f19c4)
![20240412140729_1](https://github.com/FakeFishGames/Barotrauma/assets/122838333/d35e8075-e01b-453d-b18c-0e02702ae444)

### Reproduction steps

quickstart in wastesbasic with LOS mode transparent / translucent

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.3.0.2

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This is actually intended: that's the "background sprite" of the wall, visible behind the destructible part of the sprite. If you saw the walls normally, you could use it to e.g. see where monsters are attacking the sub.
